app/controllers/v1/terms_controller.rb

```diff
@@ -96,11 +97,24 @@ def options
 
     private
 
+      def page
+        page = (params[:page].blank?) ? 1 : params[:page].to_i
+        page = 1 if page < 1
+        page
+      end
+
+      def per_page
+        per_page = (params[:per_page].blank?) ? URIService::PER_PAGE_DEFAULT : params[:per_page].to_i
+        per_page = URIService::PER_PAGE_DEFAULT if per_page < 1
+        per_page
+      end
+
       def valid_search_params?
         valid_params = [
           'action', 'controller', 'format', 'vocabulary_string_key', 'q', 'uri',
-          'authority', 'pref_label', 'alt_label', 'term_type'
+          'authority', 'pref_label', 'alt_label', 'term_type', 'per_page', 'page', 'term'
```

Comment:
This may be obvious elsewhere in the code, but can you remind me where the param 'term' is in the valid_params array?

Reply:
I was having trouble with this, it seems that rails sets to term = nil because a term is not passed in as a path param. I couldn't really find a way around this.

Reply:
Oh okay, I didn't realize that raild did that.  That's weird.  Makes sense to have it in there.  Would you mind adding a TODO comment in there so we can look into that later?  Maybe there's a way to get rid of that nil value.

Reply:
Sure